src/main/java/org/openpnp/gui/LogPanel.java

```diff
@@ -157,7 +158,16 @@ public void contentsChanged(ListDataEvent e) {
 
     private void scrollDownLogPanel() {
         if (autoScroll) {
-            logEntryJList.ensureIndexIsVisible(logEntries.getSize() - 1);
+            JScrollBar verticalBar = scrollPane.getVerticalScrollBar();
+            AdjustmentListener downScroller = new AdjustmentListener() {
+                @Override
+                public void adjustmentValueChanged(AdjustmentEvent e) {
+                    Adjustable adjustable = e.getAdjustable();
+                    adjustable.setValue(adjustable.getMaximum());
+                    verticalBar.removeAdjustmentListener(this);
+                }
+            };
+            verticalBar.addAdjustmentListener(downScroller);
```

Comment:
Won't this cause the downScroller Listener to be added to the verticalBar every time a line is added to the log?